README.md

```diff
@@ -8,6 +8,13 @@ conventional block devices. ZenFS ensures that there is no background
 garbage collection in the file system or on the disk, improving performance
 in terms of throughput, tail latencies and disk endurance.
 
+## Community
+For help or questions about zenfs usage (e.g. "how do I do X?") see below, on join us on [Matrix](https://app.element.io/#/room/#zonedstorage-general:matrix.org), or on [Slack](https://join.slack.com/t/zonedstorage/shared_invite/zt-uyfut5xe-nKajp9YRnEWqiD4X6RkTFw).
```

Comment:
The slack link is only valid for a week (IIRC). Generally, other communities would use some slack invitation service for this, or let users send email to the community member to get an invitation.

Reply:
Good point. It is a couple of months back since I created the slack link, and it looks like it still works. I think I may created a link that didn't expire.

Reply:
That's great! I just had a try, and joined Slack successfully 🤣🤣